src/lib/Stickier.js

```diff
@@ -36,8 +36,8 @@ class Stickier {
   ) {
     // Set up variables from options
     this.element = stickyElement
-    this.context = contextElement || this.element.offsetParent
-    this.container = containerElement || this.element.offsetParent
+    this.context = contextElement || this.element.parentElement
+    this.container = containerElement || this.element.parentElement
```

Comment:
Why did you chose to use [parentElement](https://developer.mozilla.org/en-US/docs/Web/API/Node/parentElement) here? It always returns the parent element, regardless if it is positioned or not (contrary to [offsetParent](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/offsetParent) which returns the nearest positioned element up the DOM hierarchy). This change will probably break the positioning logic in `_stickier()`.

Reply:
because I had an issue with offsetParent, wich resulted to undefind. But thanks for the hint. then I have to investigate deeper...

Reply:
Thanks. Found the reason on the other end. Vue3 changes the behaviour of $el which leaded to a differnet element.
revered here 89002a1434aeccc9de2801af040cdd7d989ac497